### PR TITLE
Improve the Java generation workflow

### PR DESCRIPTION
**Pull request by andreaTP**
This is so much better 🙂 cc. @kornys @henryZrncik

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -351,7 +296,13 @@
...
                 </executions>
                 <configuration>
-                    <source>${project.build.directory}/manifests</source>
+                    <urls>
+                        https://raw.githubusercontent.com/redhat-developer/app-services-operator/main/olm/olm-catalog/rhoas-operator/${rhoas-model.version}/manifests/rhoas-operator.cloudserviceaccountrequest.crd.yaml,
+                        https://raw.githubusercontent.com/redhat-developer/app-services-operator/main/olm/olm-catalog/rhoas-operator/${rhoas-model.version}/manifests/rhoas-operator.cloudservicesrequests.crd.yaml,
+                        https://raw.githubusercontent.com/redhat-developer/app-services-operator/main/olm/olm-catalog/rhoas-operator/${rhoas-model.version}/manifests/rhoas-operator.kafkaconnections.crd.yaml,
+                        https://raw.githubusercontent.com/redhat-developer/app-services-operator/main/olm/olm-catalog/rhoas-operator/${rhoas-model.version}/manifests/rhoas-operator.serviceregistryconnections.crd.yaml
+                    </urls>
+                    <downloadTarget>${basedir}/target</downloadTarget>
```
Comment (andreaTP) [inline pom.xml:305]: we missed an `mkdirs` @manusa 😅 , on it 🙂

**Review by kornys — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (manusa) [reply pom.xml:305]: 😓

**Comment by andreaTP** [comment]
@henryZrncik do you see any issue in getting this PR merged?

**Review by henryZrncik — APPROVED** [review]

**Comment by henryZrncik** [comment]
Tested on operators, runs correctly.

**Pull request merged**